dipy/reconst/shm.py

```diff
@@ -231,6 +231,60 @@ def real_sph_harm(m, n, theta, phi):
 
 
 def real_sym_sh_mrtrix(sh_order, theta, phi):
+    """
+    Compute real symmetric spherical harmonics as in Tournier 2007 [2]_, where
+    the real harmonic $Y^m_n$ is defined to be::
+
+        Real($Y^m_n$)       if m > 0
+        $Y^0_n$             if m = 0
+        Imag($Y^|m|_n$)     if m < 0
+
+    This may take scalar or array arguments. The inputs will be broadcasted
+    against each other.
+
+    Parameters
+    -----------
+    sh_order : int
+        The maximum degree or the spherical harmonic basis.
+    theta : float [0, pi]
```

Comment:
Not sure if it is the right way to write like this `theta : float [0, pi] ` in the docstring. 
I assume it means theta can take value within the range of `0-pi`.

What do you think? @skoudoro

Reply:
Based on the documentation of `real_sph_harm`. The docstring looks good.

This `theta : float [0, pi]` looks good to me.

Reply:
I confirm that `[0, pi]` should be with the description below